certbot/docs/contributing.rst

```diff
@@ -134,24 +134,20 @@ You can test your code in several ways:
 Running automated unit tests
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When you are working in a file ``foo.py``, there should also be a file ``foo_test.py``
-either in the same directory as ``foo.py`` or in the ``tests`` subdirectory
-(if there isn't, make one). While you are working on your code and tests, run
-``python foo_test.py`` to run the relevant tests.
+To run all unittests:
```

Comment:
```suggestion
To run all unittests, mypy, and lint:
```

Reply:
`tox -e lint` `tox -e mypy` `tox -e py` will run them individually, if you want to add that. or people can look inside `tox.ini` to see more info.

Reply:
If you intentionally left out the specifics that's fine too no real preference either way, this can stay as-is